rotkehlchen/tests/unit/test_cost_basis.py

```diff
@@ -406,5 +412,106 @@ def test_reduce_asset_amount_more_that_bought(accountant):
 
     # Also reduce WETH, to make sure it's counted same as ETH
     assert not cost_basis.reduce_asset_amount(A_WETH, FVal(3), 0)
-    acquisitions_num = len(asset_events.acquisitions)
+    acquisitions_num = len(asset_events.acquisitions_manager.get_acquisitions())
     assert acquisitions_num == 0, 'all buys should be used'
+
+
+def test_accounting_lifo_order(accountant):
+    asset = A_ETH
+    cost_basis = accountant.pots[0].cost_basis
+    cost_basis.reset(DBSettings(cost_basis_method=CostBasisMethod.LIFO))
+    asset_events = cost_basis.get_events(asset)
+
+    event1 = AssetAcquisitionEvent(
+        amount=FVal(1),
+        timestamp=1,
+        rate=FVal(1),
+        index=1,
+    )
+    event2 = AssetAcquisitionEvent(
+        amount=FVal(1),
+        timestamp=2,
+        rate=FVal(1),
+        index=2,
+    )
+    asset_events.acquisitions_manager.add_acquisition(event1)
+    asset_events.acquisitions_manager.add_acquisition(event2)
+    assert cost_basis.reduce_asset_amount(A_ETH, FVal(1), 0)
+    acquisitions = asset_events.acquisitions_manager.get_acquisitions()
+    assert len(acquisitions) == 1 and acquisitions[0] == event1
+
+    cost_basis.reset(DBSettings(cost_basis_method=CostBasisMethod.LIFO))
+    asset_events = cost_basis.get_events(asset)
+
+    event1 = AssetAcquisitionEvent(
```

Comment:
Why are you using event1 and event2 again here? It's confusing to read the test and understand what it's doing

Perhaps a bit of comments in the test would also make it easier to read?

Reply:
Yeah, sure